blueocean-personalization/package.json

```diff
@@ -9,7 +9,8 @@
     "test": "gulp test",
     "test:watch": "gulp test:watch",
     "bundle": "gulp bundle",
-    "bundle:watch": "gulp bundle:watch"
+    "bundle:watch": "gulp bundle:watch",
+    "postinstall": "node ../bin/cleanshrink.js"
```

Comment:
this introduces a dependency o the local node even when called from mvn front end plugin (not sure if a problem..) doesn't it?